shanghai.example.yaml

```diff
@@ -4,6 +4,7 @@ networks:
     # serves as base for each network
     # networks can overwrite any setting here.
     nick: Shanghai
+    user: Shanghai
```

Comment:
I'll let you make the changes to this since you apparently had some local changes regarding it.